src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PETypeParameterSymbol.cs

```diff
@@ -705,5 +706,12 @@ private NamedTypeSymbol GetDefaultBaseType()
         {
             get { return null; }
         }
+
+#nullable enable
+        internal bool DeriveCompilerFeatureRequiredUseSiteInfo(ref UseSiteInfo<AssemblySymbol> result, MetadataDecoder decoder)
+        {
+            PEUtilities.DeriveUseSiteInfoFromCompilerFeatureRequiredAttributes(ref result, this, (PEModuleSymbol)ContainingModule, Handle, CompilerFeatureRequiredFeatures.None, decoder);
+            return result.DiagnosticInfo != null && IsHighestPriorityUseSiteError(result.DiagnosticInfo);
```

Comment:
>IsHighestPriorityUseSiteError [](http://example.com/codeflow?start=52&length=29)

Is it valid to call this method for a type parameter? Type parameters do not have use-site errors on their own.